test/regression.uts

```diff
@@ -1049,7 +1049,7 @@ PPP('\x80!\x01\x01\x00\x10\x03\x06\xc0\xa8\x01\x01\x02\x06\x00-\x0f\x01')
 p=_
 assert(p[PPP_IPCP].code == 1)
 assert(p[PPP_IPCP_Option_IPAddress].data=="192.168.1.1")
-assert(p[PPP_IPCP_Option].data == '\x00-\x0f\x01')
+assert(p[PPP_IPCP_Option].data == "192.168.1.1")
```

Comment:
You are "cheating" here, this is not what was tested, you should write `[PPP_IPCP_Option,2]`.